src/reset-password/tests/__snapshots__/ResetPasswordPage.test.jsx.snap

```diff
@@ -147,10 +147,9 @@ exports[`ResetPasswordPage should match reset password default section snapshot
         </div>
       </div>
       <button
-        className="btn btn-primary submit"
-        onBlur={[Function]}
+        className="btn-primary submit btn btn-primary"
+        disabled={false}
```

Comment:
I think by default it would be false. so we don't need to explicitly assign it.

Reply:
This is generated by the tool.